Framework/GenAIFramework/Editor/Scripts/scene_generation/README.md

```diff
@@ -0,0 +1,27 @@
+# Scene generation feature
+
+This feature demonstrates the use of the GenAIFramework for scene generation. Language model is used for Python code generation that manipulates prefabs through the interface available in [Python Editor Bindings](https://docs.o3de.org/docs/user-guide/editor/editor-automation/).
```

Comment:
`The language model`

Reply:
I think we don't refer to a specific model so no "the" is ok.

Reply:
I have moved the readme to gettingStarted section [42d7eb5](https://github.com/RobotecAI/o3de-genai-gems/pull/158/commits/42d7eb54204beeea9d40058c5ae2bb83f12a1a0b)